go.mod

```diff
@@ -147,3 +169,8 @@ replace (
 	github.com/gogo/protobuf v1.3.0 => github.com/gogo/protobuf v1.3.2
 	github.com/gogo/protobuf v1.3.1 => github.com/gogo/protobuf v1.3.2
 )
+
+// Required by ??? TODO
+replace k8s.io/cri-api => k8s.io/cri-api v0.25.8
```

Comment:
This shouldn't be needed anymore after we update to [latest version](https://github.com/inspektor-gadget/inspektor-gadget/pull/1538). Does `containerd` complains about it?

Reply:
Not needed anymore after a rebase